components/automate-chef-io/content/docs/on-prem-builder.md

```diff
@@ -46,6 +50,35 @@ Roughly 80 GB of the disk space is designated for Chef Automate; the rest is use
 Chef Habitat Builder and the artifacts it stores. The current implementation uses Minio for
 Chef Habitat artifact storage; we do not support using Artifactory for artifact storage.
 
+### Hardware Sizing Considerations
+
+#### Environment Variables
+The frequency at which Habitat Supervisors poll the on-premises Chef Builder API for changes can be controlled by three Environment variables in the Chef Habitat Supervisor's runtime.
+
+| Variable | Default | Description |
+| --- | --- | --- |
+| `HAB_SUP_UPDATE_MS` | 60000 | Frequency in milliseconds governing how often to check for Supervisor updates when running with the [--auto-update](https://www.habitat.sh/docs/habitat-cli/#hab-sup-run) flag. |
+| `HAB_UPDATE_STRATEGY_FREQUENCY_MS` | 60000 | Frequency in milliseconds governing how often to check for service updates when running with an [update strategy](https://www.habitat.sh/docs/using-habitat/#using-updates) |
+| `HAB_UPDATE_STRATEGY_FREQUENCY_BYPASS_CHECK` | no default | If set to `1`, will enable the Supervisor to utilize the `HAB_UPDATE_STRATEGY_FREQUENCY_MS` variable value from the line above for service updates. |
+
+These variables greatly influence the polling rate that the Chef Builder API will have to handle.
+
+#### Planning Guide
+The planning guide can be used for appropriately sizing hardware at differing client workloads.
+
+* `Clients` refers to the total number of Chef Habitat Supervisors running with the `--auto-update` flag plus the total number of Service Instances being managed with an update strategy.
+* `Interval` refers to the value of `HAB_SUP_UPDATE_MS` and `HAB_UPDATE_STRATEGY_FREQUENCY_MS` and assumes they are both set to the same value.
+
```

Comment:
It is implicit in the table below, but it might be worth calling out that the configs below support a certain update API call rate, and it's the client_count/update_frequency tradeoff that's being made. I think you showed some formulas for that recently, I found them helpful.
If the table below represents tested configurations, that might be worth highlighting.